server/index.ts

```diff
@@ -1,10 +1,9 @@
 import { config } from 'dotenv-flow';
+config({ silent: false });
```

Comment:
the other imports (notably `./server` was attempting to access environment variables before they were loaded